kong/pdk/request.lua

```diff
@@ -568,10 +569,8 @@ local function new(self)
       error("header name must be a string", 2)
     end
 
-    local header_value = _REQUEST.get_headers()[name]
-    if type(header_value) == "table" then
-      return header_value[1]
-    end
+    -- Do not localize ngx.re.gsub! It will crash because ngx.re is monkey patched.
```

Comment:
I am not sure about it, but I found we do localize `ngx.re.gsub`, such as in `router.lua`.